src/rgw/rgw_op.cc

```diff
@@ -6,6 +6,8 @@
 
 #include <sstream>
 
+#include <boost/optional.hpp>
+
```

Comment:
I think we need an extra `#include <boost/utility/in_place_factory.hpp>` here for `boost::in_place()` - see https://github.com/ceph/ceph/pull/9817